modules/sdk/gradle-plugins-rest-builder/src/main/java/com/liferay/gradle/plugins/rest/builder/BuildRESTTask.java

```diff
@@ -126,24 +136,18 @@ private List<String> _getCompleteArgs() {
 			args.add(forcePredictableOperationId);
 		}
 
-		if (_isJakartaEnabled()) {
+		if (_jakartaEnabled) {
 			args.add("--javaee-package");
 			args.add("jakarta");
 		}
 
 		return args;
 	}
 
-	private boolean _isJakartaEnabled() {
-
-		// TODO Return a value based on the branch or workspace
-
-		return false;
-	}
-
 	private Object _copyrightFile;
 	private Object _forceClientVersionDescription;
 	private Object _forcePredictableOperationId;
+	private boolean _jakartaEnabled;
```

Comment:
This needs to follow the pattern of:
- field is an `Object`
- `setter` accepts any type (`Object`)
- `getter` returns specific type (`boolean`)
This maintains Gradle's flexibility in how you set values. It could be you're setting a boolean value directly, or it could be you are passing in a Closure/function to be invoked later/lazily that will provide a boolean as late as possible.